pkg/filepos/position.go

```diff
@@ -39,6 +39,10 @@ func (p *Position) Line() int {
 	return *p.line
 }
 
+func (p *Position) FileString() string {
```

Comment:
"FileString" felt ambiguous, maybe an alternative like FileName describes what happens in this function better?

Reply:
Removed entirely :)
It isn't the file that matters, it's the appropriately annotated document. Went back to using AsCompactString().